dev/spec-kitty/kittify/scripts/dashboard/static/dashboard/dashboard.css

```
@@ -0,0 +1,749 @@
+}
+
+.header-info {
+    display: flex;
+    flex-direction: column;
+    gap: 6px;
+}
+
+.tree-view {
+    font-size: 0.75em;
+    color: var(--medium-text);
+    font-family: 'Monaco', 'Menlo', monospace;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)